app/controllers/posts_controller.rb

```diff
@@ -27,6 +27,20 @@ def create
     end
   end
 
```

Comment:
- [ ] Kindly use **CanCanCan** for Post authorization in `Post Controller`, something like so:

`load_and_authorize_resource`